### PR TITLE
download rollback jenkins commits

### PR DESCRIPTION
**Pull request by divyagandhi01**
Jenkins commits should be downloaded even for rollback releases.

### CODE REVIEW

**Comment by TingluoHuang** [comment]
@GitHubSriramB

File: src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs [hunk 1]
```diff
@@ -118,10 +118,10 @@ public class JenkinsArtifact : AgentService, IArtifactExtension
                         else if (startJobId > endJobId)
                         {
                             context.Output(StringUtil.Loc("JenkinsRollbackDeployment"));
-
-                            // we do not support fetching the commits for the rollback deployment fully yet.
-                            // until then we will return from here.
-                            return;
+                            int swap=startJobId;
```
Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs]: space before and after =

File: src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs [hunk 2]
```diff
@@ -118,10 +118,10 @@ public class JenkinsArtifact : AgentService, IArtifactExtension
...
+                            startJobId=endJobId;
+                            endJobId=swap;
+                            //swap the job IDs to fetch the roll back commits
```
Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs]: move comment above, space after //

**Comment by GitHubSriramB** [comment]
Update JenkinsRollbackCommitsNotSupported L0 test given that we are now supporting rollback commits

File: src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs [hunk 3]
```diff
@@ -118,10 +118,10 @@ public class JenkinsArtifact : AgentService, IArtifactExtension
                         else if (startJobId > endJobId)
                         {
                             context.Output(StringUtil.Loc("JenkinsRollbackDeployment"));
```
Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs]: add the start & end job id to the output string. see how it is done for DownloadingJenkinsCommitsBetween

**Pull request merged**